src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -2796,12 +2796,11 @@ public Dataset GetCompatibleDataset(RoleMappedData data, PredictionKind kind, in
     }
 
     public abstract class TreeEnsembleModelParameters :
-        PredictorBase<Float>,
+        ModelParametersBase<Float>,
         IValueMapper,
         ICanSaveInTextFormat,
         ICanSaveInIniFormat,
         ICanSaveInSourceCode,
-        ICanSaveModel,
```

Comment:
>ICanSaveModel [](start = 8, length = 13)

did u mean this?

Reply:
Yes, the base class now has `ICanSaveModel`

---
In reply to: [242426241](https://github.com/dotnet/machinelearning/pull/1901#discussion_r242426241) [](ancestors = 242426241)